awspub/context.py

```diff
@@ -64,19 +65,28 @@ def source_sha256(self):
         return self._source_sha256
 
     @property
-    def tags(self):
+    def tags_dict(self) -> Dict[str, str]:
```

Comment:
this should return a typeddict and then just name the method appropriatley`. no need to change this now though